### PR TITLE
Greeter

### PR DESCRIPTION
**Pull request by emelieedman**
Hey guys! I'm done with the greeter if you wanna have a look and give feedback! It greets you depending on what time of day it is (morning, afternoon & night). I also added the styling from the figma file. To see the changes you have to switch to that branch by typing: 'git checkout greeter' in the terminal & then 'git pull' to pull down the changes from GitHub.

### CODE REVIEW

**Comment by asaback** [comment]
Nice! 
Only thing is maybe we should the component in a separate folder?

**Comment by essisuomalainen** [comment]
Looking good!

File: src/Greeter.js [hunk 1]
```diff
@@ -6,15 +6,15 @@ const Greeter = () => {
...
-    timeOfDay = "afternoon";
+    timeOfDay = "AFTERNOON";
   } else {
-    timeOfDay = "night";
+    timeOfDay = "NIGHT";
   }
   return (
     <h1 className="greeter-text">
-      Good {timeOfDay} <br></br> sunshine!
+      GOOD {timeOfDay} <br></br> SUNSHINE!
```
Comment (emelies) [inline src/Greeter.js]: Could maybe do "sunshine" for morning and another word for night and afternoon, maybe would look and feel better UX-wise

**Comment by emelies** [comment]
Great job! the code looks clean and rhymes with the design! :D

File: src/Greeter.js (hunk 1 above)
Comment (emelieedman) [reply src/Greeter.js]: I was thinking of changing it so instead of "sunshine" it greets you with your name instead! :)

**Pull request merged**